### PR TITLE
fix: Update tags for extension to enable better prioritization

### PR DESCRIPTION
**Pull request by mathur07**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
...
   ],
   "keywords": [
-    "dependencies",
+    "security",
     "vulnerability",
-    "license",
+    "dependency scan",
     "node",
     "java",
-    "multi-root ready"
```
Comment (arajkumar) [inline package.json:38]: keep "multi-root ready", IIRC this would enable keywords to be pickedup from `contributes.languages[*].aliases`

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mathur07) [reply package.json:38]: Okay :+1

Comment (mathur07) [reply package.json:38]: Okay :+1:

Comment (mathur07) [reply package.json:38]: @arajkumar this is something different right?
https://code.visualstudio.com/docs/editor/multi-root-workspaces

**Review by arajkumar — APPROVED** [review]
lgtm

**Review by dvandra — APPROVED** [review]
LGTM

**Comment by dvandra** [comment]
[test]

**Pull request merged**